### DISCUSSION TITLE
Pregunta 3 examen 2018-2 (Greedy)

### DISCUSSION THREAD

**Discussion by Raxtesh** [IIC2133-PUC/2024-2#181] [Preguntas Materia]
Tengo una duda, porque si esta demostrando por contradiccion, y define 2 conjuntos (S y R) con los puntos validos, siendo |S| < |R|, se infiere que  los puntos de R están más lejos que los puntos de S; es decir, para cada j = 1, 2, …, m, tenemos que x_pj ≥ x_qj?

**Comment by tomastrivino** [comment]
Hola,
sorry la demora. Para la próxima porfa adjunta foto del ejercicio para cachar bien el contexto!
